guide/sharding/README.md

```diff
@@ -40,7 +44,7 @@ client.on('message', message => {
 client.login('token');
 ```
 
-Let's say your bot is in a total of 3,600 guilds. With the recommended shard count, you might be in 4 shards, the first 3 containing 1,000 guilds each, and the last one containing the remaining 600. If a guild on a certain shard (shard #2, for example) and it receives this command, the guild count will be 1,000, which is obviously not the "correct" number of guilds for your bot. Likewise, if the message is received on a guild in shard 3, (shards go up from 0 to n), the guild count will be 600, which is still not what you want. "How can I fix this?", you ask? Well, that's why we're here, isn't it?
+Let's say your bot is in a total of 3,600 guilds. With the recommended shard count, you might be in 4 shards, the first 3 containing 1,000 guilds each, and the last one containing the remaining 600. If a guild on a certain shard (shard #2, for example) and it receives this command, the guild count will be 1,000, which is obviously not the "correct" number of guilds for your bot. Likewise, if the message is received on a guild in shard 3, (shard IDs are zero-indexed), the guild count will be 600, which is still not what you want. "How can I fix this?", you ask? Well, that's why we're here, isn't it?
```

Comment:
Also not directly related to your change, but the second sentences feels a bit awkward to me (again, because of the constant comma use). Instead of a comma after "you might be in 4 shards", I think it could be replaced with an em dash.